node/components.go

```diff
@@ -26,6 +27,18 @@ func bridgeComponents(cfg *Config, store Store) fxutil.Option {
 		fxutil.Supply(Bridge),
 		baseComponents(cfg, store),
 		nodecore.Components(cfg.Core, store.Core),
+		fxutil.Provide(services.LightAvailability), // TODO(@Wondertan): Remove strict requirements to have Availability
```

Comment:
This requires us to change `ShareService` not to take `Availability` right?

Reply:
Yes